### PR TITLE
Don't default to showing 'trusted' in /userauth

### PR DESCRIPTION
**Pull request by AnnikaCodes**

### CODE REVIEW

File: server/chat-commands/moderation.ts [hunk 1]
```diff
@@ -286,9 +286,9 @@ export const commands: ChatCommands = {
 		const buffer = [];
 		let innerBuffer = [];
 		const group = Users.globalAuth.get(targetId);
-		if (group) {
-			buffer.push(`Global auth: ${group === ' ' ? 'trusted' : group}`);
-		}
+		let showTrusted = false;
```
Comment (scheibo) [inline server/chat-commands/moderation.ts:289]:
```suggestion
const showTrusted = Users.isTrusted(targetID) && group === ' ';
```

File: server/chat-commands/moderation.ts [hunk 2]
```diff
@@ -286,9 +286,9 @@ export const commands: ChatCommands = {
...
+		if (group === ' ' && Users.isTrusted(targetId)) showTrusted = true;
```
Comment (scheibo) [inline server/chat-commands/moderation.ts:290]:
```suggestion
const showTrusted = group === ' ' && Users.isTrusted(targetID);
```

File: server/chat-commands/moderation.ts [hunk 3]
```diff
@@ -286,9 +286,8 @@ export const commands: ChatCommands = {
 		const buffer = [];
 		let innerBuffer = [];
 		const group = Users.globalAuth.get(targetId);
-		if (group) {
-			buffer.push(`Global auth: ${group === ' ' ? 'trusted' : group}`);
-		}
+		const showTrusted = group === ' ' && Users.isTrusted(targetID);
```
Comment (scheibo) [inline server/chat-commands/moderation.ts:289]:
```suggestion
		const showTrusted = group === ' ' && Users.isTrusted(targetId);
```

File: server/chat-commands/moderation.ts [hunk 4]
```diff
@@ -286,9 +286,8 @@ export const commands: ChatCommands = {
 		const buffer = [];
 		let innerBuffer = [];
 		const group = Users.globalAuth.get(targetId);
-		if (group) {
-			buffer.push(`Global auth: ${group === ' ' ? 'trusted' : group}`);
-		}
+		const showTrusted = group === ' ' && Users.isTrusted(targetId);
+		if (group !== ' ' || showTrusted) buffer.push(`Global auth: ${showTrusted ? 'trusted' : group}`);
```
Comment (Zarel) [inline server/chat-commands/moderation.ts:290]:
```suggestion
```suggestion
		if (group !== ' ' || Users.isTrusted(targetId)) {
			buffer.push(`Global auth: ${group === ' ' ? 'trusted' : group}`);
		}
``````

File: server/chat-commands/moderation.ts [hunk 5]
```diff
@@ -286,7 +286,7 @@ export const commands: ChatCommands = {
 		const buffer = [];
 		let innerBuffer = [];
 		const group = Users.globalAuth.get(targetId);
-		if (group) {
+		if (group !== ' ' || Users.isTrusted(targetId)) {
```
Comment (Zarel) [inline server/chat-commands/moderation.ts:289]:
```suggestion
		if (Users.globalAuth.has(targetId)) {
```

Comment (Zarel) [reply server/chat-commands/moderation.ts:289]: n-nothing wrong with your code; I'm just golfing

Comment (AnnikaCodes) [reply server/chat-commands/moderation.ts:289]: yeah, it looks like trusted users who aren't global auth aren't in `Users.globalAuth`

**Pull request merged**